src/App.tsx

```diff
@@ -1,6 +1,62 @@
-import React from 'react';
+import React, { useState } from 'react';
+import 'bootstrap/dist/css/bootstrap.min.css';
 import './App.css';
+import { Pagination } from './Pagination';
+import { Content } from './Content';
 
-const App: React.FC = () => <h1>Pagination</h1>;
+const App: React.FC = () => {
+  const amount = 42;
+  const pageContent = [];
+  const [itemsPerPage, setItemsPerPage] = useState(5);
+  const [selectedPage, setSelectedPage] = useState(1);
+  const [selectedItemsPerPage, setSelectedItemPerPage] = useState(5);
+
+  for (let i = 1; i <= amount; i += 1) {
+    pageContent.push(`Content item ${i}`);
+  }
```

Comment:
Will be better to create separate array with content data outside of the App component